src/App.tsx

```diff
@@ -1,23 +1,22 @@
 import React from 'react';
 
-import styled, { ThemeProvider } from "styled-components"
+import { ThemeProvider } from "styled-components"
 import { GlobalStyles } from './styles/Global'
 import { theme } from "./styles/Theme";
 
 import { Sidebar } from './components/Sidebar/Sidebar';
 import { Content } from "./components/Content/Content";
 
 const App: React.FC = () => {
-  return (
-    <ThemeProvider theme={theme}>
-      <GlobalStyles />
-      <div className="App">
-        <Sidebar />
-        <Content />
-      </div>
-    </ThemeProvider>
-  )
-
+    return (
```

Comment:
@ArtsemiLasyi Did you use some Linter or this is your own code style?

Reply:
Yep, my own style code